src/Workspaces/Core/Portable/Storage/SQLite/v2/Interop/SqlConnection.cs

```diff
@@ -243,6 +244,8 @@ public ResettableSqlStatement GetResettableStatement(string query)
             }
             catch (SqlException ex)
             {
+                Logger.Log(FunctionId.SQLite_SqlException, SQLitePersistentStorage.GetLogMessage(ex));
```

Comment:
based on the conversation yesterday I think you need to pass `LogLevel.Information` here in order for this to be reported to telemetry.
https://sourceroslyn.io/#Microsoft.CodeAnalysis.Remote.ServiceHub/TelemetryLogger.cs,158

Reply:
Yup! And why we're discussing making this more clear :)